Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html

```diff
@@ -0,0 +1,75 @@
+Assignment Part One: Object-Property Pair contradictions.
+
+"PROVIDE OWL-2 DIRECT SEMANTIC BASED EXPLANATIONS
+FOR UNSATISFIABILITY AND INCONSISTENCY RESULTS
+STEMMING FROM ROLE CONSTRAINT PAIRS."
+
+
+1.) FUNCTIONAL COMBINED WITH TRANSITIVE BECOMES UNDECIDEABLE
+
+According to functional object property, if xRy and xRz then y = z 
+
+By transitive, if xRy, yRz, then xRz where x,y and z are distinct.
+
+xRy and xRz cannot hold simultaneously because this violates cardinality of functional object properties. 
+
```

Comment:
I don't think this is how we were supposed to do assignment 1. Let's talk tomorrow!